docs/notebooks/translation_matrix.ipynb

```diff
@@ -0,0 +1,1352 @@
QG/6j/9I3v/+nuCa\nJK95zQvHHHVUzz/Hjk2WLOnZvvnmnjCaJHvumYwe/ez4jS3rvuGGpLv72fcffzx58snkla9s/ToA\nepnwCgDQNFtv3fPPgQOTVas2PGb9wLr+zOvy5c8dc/vtyVZbtb9GgF7mnlcAgN70jnckV12VPPRQ\nz+uHH960zx14YDJjRs/2z36WzJ377Hs77JDMn5+sWZN861vP7j/kkOcuSZ4zp7XaATrIzCsAQG/a\nfffk//v/koMOSgYN6nloU1fXs+8///7VZ5x0Us+DmN7ylmTUqGTffZ997x//MTn88GS77Xr2P/54\nz/7zzkv+4i96Hg61enVPAJ42bbNdGsDm5KtyAOizfFVOe+jNDTVpUnLOOcmYMZ2uBGCT+aocAID+\nZmMztAB9lJlXAPosM6/toTcD0C5mXgEAAOjThFcAAAAaT3gFAACg8YRXAIDN6H+cdlr22nXX/Oxn\nP+t0KQBbNN/zCgCwmTzyyCP54pe+lMdXrMiqVas6XQ7AFs3MKwDAZvK/L7ooj69YkUn77pu99967\n0+UAbNF8VQ4AfZavymkPvfnlWblyZXZ9wxuy9He/y3e+850cfvjhnS4JoON8VQ4AQMNcfdVVWfq7\n32XkG9+Yww47rNPlAGzxhFcAgDarteacM89Mkpz2mc9kwAB/5QJolWXDAPRZlg23h978h7v55ptz\n0EEHZfiwYVnym99kyJAhnS4JoBEsGwYAaIi77rorBx10UJLkQx/5iOAK0CbCKwBAG331wgtzbCl5\nw8CBWdPpYgD6EMuGAeizLBtuD715061cuTI7ve51ueKxx3L0kCGZu3Bhdtppp06XBdAYlg0DADTA\njTfemDcn+dchQ3Lypz4luAK0kZlXAPosM6/toTdvug//l/+SAf/+7/n+q1+dBUuXZtiwYZ0uCaBR\nWunNg9pdDABAf/Tkk0/mmu99LzttvXU+e9ZZgitAm1k2DADQBt/97nfz6PLlWT18eD5+wgmdLgeg\nzxFeAQDa4NJp05IkX7jggmy11Va56KKLctlll3W4KoC+wz2vAPRZ7nltD735pT388MN57Wtfm713\n2y2zu7tTin/tADbE04YBADro0ksuSZI8vnp1xowZk+OPPz5nnnlmpk6dmiSZNGlSTj/99EyYMCEj\nR47Mj3/84yTJ8uXLM3ny5LzlLW/JUUcdlf322y+zZ89OkufcM/vNb34zH/nIR5Ikv/3tb3P00Udn\nwoQJmTBhQm699dbevFSAjmnpgU2llG2TXJFkRJLFSf5rrfXRDYx7dZL/nWSPJGuSfLTWensr5wYA\naIq/+m//LSXJHXfckW233TaPPPJIzjvvvOeMWb16dW6//fZcd911mTJlSq6//vpceOGFGTp0aH7+\n85/nnnvuydixY9eNf/7s7TOvTz311Jx22mnZf//9s3Tp0hx66KG59957N/s1AnRaq08bPj3JDbXW\nL5RSPp3kjLX7nu+8JN+rtb6/lDIoyStbPC8AQCM88MADSZITP/nJbLvttkmS17zmNS8Yd9RRRyVJ\nxo4dmyVLliRJbr755px66qlJkj333DOjR49eN35jS7VvuOGGdHd3r3v/8ccfz5NPPplXvtJfr4C+\nrdXwekSSg9Zu///t3X+sVGV6wPHvIz8s64phFUT8hZvWFSQKFFmatQJutFCt2mT9sTUtbI00rKJt\nbOpubSP8UxetiS6sRg3t0ubu6oYNlbrdrK56TYxdQVm5Yu+CG4OrVilGkeCPVvHpHzNe4Xrn3uHO\ncOfcOd9PMmF+vHfmed/zzjw8c86cdy3QSa/iNSLGAr+fmYsBMvMjYE+DrytJktRy3d3dPPnkk1x5\n5ZUcc8wx/bY9/PDDARgxYgQfffRRn232L1j33/P6wQcfHNDm6aefZtSoUY2ELknDTqO/eZ2QmTsB\nMvMNYEIfbU4B3oyIf46IzRFxb0SMafB1JUmSWmrZsr9k6tTfZcmSf6SjYx2rV3+Pt956C6icwKke\n55xzDh0dHQBs3bqVrq6unscmTpzItm3b+Pjjj1m/fn3P/eeff/4BhyRv2bKlGd2RpMIbsHiNiEci\nomu/y/PVfy/qo3lfx7eMBGYC38vMmcB79H1osSRJ0rDQ3d3N6tX3Ar8AtgEb2b17L3PmzGHGjBnc\ncMMNB+w5rXX24aVLl7J3715OP/10li9fzqxZs3oeu+WWW7jgggs4++yzmTRpUs/9d955J8888wxn\nnnkm06ZN45577jlEvZSkYmloqZyI6AbmZebOiJgIPJ6ZU3q1ORb4z8z8YvX22cCNmflHNZ4zb775\n5p7b8+bNY968eYOOUZJUHp2dnXR2dvbcXrFihUvlNIFL5XzW2rVrWbz4H6gUrp84le9//yYWLVo0\n6OedP38+t99+OzNnzmw4RkkqokaWymn0N68bgMXASmAR8GDvBtXC9pWIODUztwNfBfo9Jd7y5csb\nDEuSVEa9v/BcsWJF64JRW5s9ezbwCtAFnFH999Xq/YPn+rCSVFuje16/APwIOBF4mcpSObsj4jjg\nvsy8sNruTCpL5YwCXgK+0deSOtW2frsrSWqKRr7d1afMzX1btux6Vq++DzgBeJVrr72aVavuHOjP\nJKnUGsnNDRWvh4IJUpLULBavzWFurq27u5uNGzcye/ZspkyZMvAfSFLJWbxKktQHi9fmMDdLkpql\nkdzc6FI5kiRJkiQdchavkiRJkqTCs3iVJEmSJBWexaskSZIkqfAsXiVJkiRJhWfxKkmSJEkqPItX\nSZIkSVLhWbxKkiRJkgrP4lWSJEmSVHgWr5IkSZKkwrN4lSRJkiQVnsWrJEmSJKnwLF4lSZIkSYVn\n8SpJkiRJKjyLV0mSJElS4Vm8SpIkSZIKz+JVkiRJklR4Fq9qqVWrVjF16lSOPvpobr311laHI0mS\nJKmgIjNbHcMBIiKLFp
```

Comment:
E226 missing whitespace around arithmetic operator (here and anywhere)

Reply:
Done